contracts/Reserve.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.3;
+pragma experimental ABIEncoderV2;
+import "./iBEP20.sol";
+interface iBASE {
+    function DAO() external view returns (iDAO);
+    function transferTo(address, uint256 ) external payable returns(bool);
+}
+interface iDAO {
+    function DAO() external view returns (address);
+}
+
+contract Reserve {
+    address public BASE;
+    address public ROUTER;
+    address public LEND;
+    address public SYNTHVAULT;
+    address public DEPLOYER;
+    bool public emissions;
+
+
+    // Only DAO can execute
+    modifier onlyGrantor() {
+        require(msg.sender == _DAO().DAO() || msg.sender == ROUTER || msg.sender == DEPLOYER || msg.sender == LEND || msg.sender == SYNTHVAULT, "Must be DAO");
+        _; 
+    }
```

Comment:
Permissions function to ensure only the parent contracts / DAO can handle certain functions.